src/libraries/System.Text.Json/tests/Serialization/StreamTests.cs

```diff
@@ -0,0 +1,511 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Follow up to https://github.com/dotnet/runtime/pull/42158#discussion_r487553719 (to raise visibility, since that is resolved)

@layomia, @devsko - what do you think?

> Since we have other code paths to allow "partial progress"/continuation via the `Utf8JsonReader` overload, something a bit more generic as the filename that covers more uses, might actually be better than using `StreamTests`. 

> Something like `ContinuationTests` seems reasonable, but **only** if it makes sense to add test cases for other overloads to this file too. See https://github.com/dotnet/runtime/pull/42158#discussion_r488907272

Reply:
I already started to work on a follow up PR for the other continuations. I'm happy to change the name then. Hope I get it done this week because I'm away some time starting next week 😁

Reply:
`ContinuationTests` is fine with me.

Reply:
Do you want to merge this PR into 5?

Reply:
Sorry I didn't see your above response. Yes we plan to port this PR to .NET 5. If Ship-room gives the OK, we should be able to port it this week. I have some additional feedback that will need another commit, so I'd appreciate doing the rename now.

> Hope I get it done this week because I'm away some time starting next week 😁

Nice, enjoy!

Reply:
> I have some additional feedback that will need another commit, so I'd appreciate doing the rename now.

Okay I wait for your feedback and will address it tomorrow (it's 8pm here)

Reply:
https://github.com/dotnet/runtime/pull/42158#discussion_r489633879